pkg/controller/deployment/deployment_controller.go

```diff
@@ -150,8 +150,8 @@ func (dc *DeploymentController) Run(workers int, stopCh <-chan struct{}) {
 	defer utilruntime.HandleCrash()
 	defer dc.queue.ShutDown()
 
-	klog.Infof("Starting deployment controller")
-	defer klog.Infof("Shutting down deployment controller")
+	klog.InfoS("Starting controller", "controller", "deployment")
+	defer klog.InfoS("Shutting down controller", "controller", "deployment")
```

Comment:
I don't understand the above changes, there are no variables here so I'd leave it as is.

Reply:
We don't keep it in a variable, but "deployment" is a controller name, isn't it?

If it was a pod, we would convert something like `"Starting pod ABC"` to `"Starting pod", pod="ABC"`.
I apply the same logic to the controller name. 

Also, If we don't have any keys, the only way to find when did the deployment controller start is to use full text search and look for "Starting deployment  controller". While specifying `controller=deployment` in a log viewer gives us a quick way to get this information.

Reply:
That's fair.